packages/components/src/styles/components/side-nav/header.scss

```diff
@@ -69,19 +69,12 @@
 }
 
 // Dropdown & Button color theming overrides
-.hds-side-nav__dropdown, // deprecated classname
```

Comment:
TODO: Test that removal of this classname doesn't affect visual appearance in consumer code bases